src/components/shared/Snackbar/index.tsx

```diff
@@ -9,30 +9,56 @@ const Container = styled.View`
   font-size: 16;
 `;
 
+interface SnackbarStyles {
+  container?: ViewStyle;
+  message?: TextStyle;
+}
+
 export interface SnackbarProps {
   text: string;
   testID?: string;
+  style?: SnackbarStyles;
 }
 
+const getStyles = (props: SnackbarProps): SnackbarStyles => {
+  const snackbar = {
+    container: {
+      padding: 16,
+      bottom: 50,
+      backgroundColor: '#303235',
+      borderRadius: 10,
+      boxShadow: '0px 11px 14px -16px rgba(61,61,61,1)',
+    },
+    message: {
+      color: '#ffffff',
+    },
+  };
+
+  return {
+    container: StyleSheet.flatten([snackbar.container, props.style.container]),
+    message: StyleSheet.flatten([snackbar.message, props.style.message]),
+  };
+};
```

Comment:
I like this new approach but I also feel bit redundant since you are already using `styled-components` which has styles in it. `container: { ...` can go inside `Container` in line 6!

Reply:
yes, I referred [this code](https://github.com/xotahal/react-native-material-ui/blob/master/src/Snackbar/Snackbar.react.js).
I agree with you, it seems redundant. 
How about removing styled-component? Because I made this PR to allow to overwrite default style with user customed styles. 
As I know, it's impossible using styled-component.

Reply:
You can still pass the styles insiss the StyledComponent that will overlap its properties!

```
Container
  style={...userStlyles}
```

Reply:
It works and makes the code simple. thanks